src/core/dev/cq_mgr_tx.cpp

```diff
@@ -180,6 +180,10 @@ int cq_mgr_tx::poll_and_process_element_tx(uint64_t *p_cq_poll_sn)
 {
     cq_logfuncall("");
 
+    if (safe_mce_sys().doca_tx) {
```

Comment:
Need to move this code from CQ Mgr to ring. So it will be easier to remove CQ mgr later.

Reply:
because we call "poll_and_process_element_tx" from many places, I put also a check inside to be on the safe side. later when we use doca only - it will be removed anyway.